conda/base/constants.py

```diff
@@ -228,6 +228,9 @@ def __str__(self):
 PACKAGE_CACHE_MAGIC_FILE = 'urls.txt'
 PREFIX_MAGIC_FILE = join('conda-meta', 'history')
 
+PREFIX_SATE_FILE = join('conda-meta', 'state.json')
+PACKAGE_ENV_VARS_DIR = join('etc', 'conda', 'env_vars')
```

Comment:
Maybe changes this to `env_vars.d` to make the .d directory pattern explicit.  It'll also then be similar to `activate.d` and `deactivate.d` in the same directory.